apps/shooting-clubs/acceptance/features/second-person-name.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('Second person name step');
+
+Before((
+  I,
+  secondPersonNamePage
+) => {
+  I.visitPage(secondPersonNamePage, steps);
+});
+
+Scenario('The correct form element are present', (
+  I,
+  secondPersonNamePage
+) => {
+  I.seeElement('#second-person-name');
```

Comment:
You should put this in the page helper and then do 
`I.seeElement(secondPersonNamePage['second-person-input']);` or something like that.